keystore/serialization/keystore.proto

```diff
@@ -15,4 +15,14 @@ message Key {
   int64 created_at = 3;
   string key_type = 4;
   bytes metadata = 5;
+}
+
+// ECDH encryption envelope for hybrid encryption
+message EcdhEnvelope {
```

Comment:
Should these be caps per go naming standards?
```suggestion
message ECDHEnvelope {
```

Reply:
sure will change